Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -582,8 +585,6 @@ open class MapView: UIView {
     open override func didMoveToWindow() {
         super.didMoveToWindow()
 
-        unsubscribeFromLifecycleNotifications()
```

Comment:
Was this the only place this method was called? If so, let's remove the method.

Reply:
done, I also removed test that related to this

Reply:
done, I also removed test that related to this

Reply:
Looks like the change to remove the method was not pushed.